standard.tex

```diff
@@ -98,54 +107,42 @@ \subsection{PCB Design}
+\item All switches and connectors need idiot-proof labeling. Labeling should be permanent (i.e. not on tape, etc).
+\item All mission critical components should have a hotswappable replacement. You should have that replacement's location written down. "Hotswappable" means you can replace it without any powered tools.
 \item If it's conceivably possible for a component to be installed backwards, there should be idiot proof documentation of which way it goes, that documentation should be less than 2 inches from the install point.
-\item Buy spares. You will fuck things up, be prepared.
-\item Don't solder at hotter than 700\degree. Even that is probably pretty aggressive. Use leaded solder, it melts at a low temperature so you won't burn off all your flux. If your solder isn't melting below 700\degree, the iron's tip is probably bad and should be replaced. Take care of your tips!
-\item Don't short things. If you have any bare wires flailing around, don't apply power to the system.
-\item All battery connections should be terminated in a female connector to prevent battery shorting and enable easy battery swapping.
-\item Before ordering any components, make a list of all components in your design, and go through their data sheets one by one and ensure that they meet or exceed the requirements of your design. Do not just trust the Digikey table values, they are occasionally mislabeled.
+\item Buy spares for all components provided it is not prohibitively expensive.
+\item All battery connectors should be terminated in a female connector to prevent battery shorting and enable easy battery swapping.
+\item Before ordering any components, make a list of all components in your design, and go through their data sheets one by one and ensure that they meet or exceed the requirements of your design. Do not just trust the Digi-Key table values, they are occasionally mislabeled. Design reviewers should also do this.
 \end{itemize}
-%end of mescellaneous subsection -- }}}
-%end of hard rules section -- }}}
+%-- }}}
+%-- }}}
 
-%start of documentation section -- {{{
+%Documentation -- {{{
 \section{Documentation Standards}
 It is your responsibility to ensure that your project is fully documented and available to the team. Development and documentation for all electrical projects should be tracked in git and hosted on GitHub (under the team organization, waterloo-rocketry). If you manage your repository using issues and pull requests, and if decisions regarding these are made in Slack conversations, put a permalink to the conversation in the issue comment (this is to help traceability). In general, how you manage your project repository is up to you.
 
-Release documentation for each project should also be put in Electrical/$\langle$project folder$\rangle$ on the Drive. Project documentation should include the following:
+Project documentation should include the following:
```

Comment:
Do we say where it should live? If we don't specify "in the git repo", it might just be posted to slack (which I think is bad)

Reply:
>Development and documentation for all electrical projects should be tracked in git and hosted on GitHub (under the team organization, waterloo-rocketry)

(see line above)